packages/connect-extension-protocol/src/index.ts

```diff
@@ -23,6 +23,8 @@
 export interface ToApplication {
   /** origin is used to determine which side sent the message **/
   origin: "content-script"
+  /** The uniqueId for extension multiplexing **/
```

Comment:
```suggestion
  /** Which chain this message applies to **/
```

Reply:
yeah, sorry, that was a copy-pasta from the `chainId` of the `ToExtension`